lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -36,10 +36,10 @@ export async function getChangeLogJSON(
   const versioning = config.versioning!;
   const currentVersion = config.currentVersion!;
   const newVersion = config.newVersion!;
-  const sourceUrl = config.sourceUrl!;
+  const sourceUrl = config.customChangelogUrl ?? config.sourceUrl!;
```

Comment:
should we pass the resolved source URL as part of function params so that we don't need to repeat this `??` logic?

Reply:
yes, please pass it from `getChangeLogJSON`